deliverables/README.md

```diff
@@ -8,61 +10,122 @@ To answer questions, please just put the answer on the next line after the quest
+
+
 
----
 ### Section 3: Git and Github  
-What is Git?  
-What is SCM?  
-What is a VCS?  
-Why is Git useful for a developer?  
-Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
-How do you create it on Github?  
-How do you commit changes?  
-What is the difference between staging and committing changes?  
-What is the difference between committing your changes and pushing them to Github?  
-What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
-What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
-What is "Cloning" a repo?  
+**What is Git?**  <br/>It is an open source program for tracking changes in text files. It was written by the author of the Linux operating system, and is the core technology that GitHub, the social and user interface, is built on top of.
+
+
+**What is SCM?** <br/>Supply chain management software, is the software tools or modules used in executing supply chain transactions, managing supplier relationships and controlling associated business processes. 
+
+
+**What is a VCS?**  <br/>Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later.
+
+
+**Why is Git useful for a developer?**  <br/>It gives him a place where he can push things up to his or her live site. 
+
+
+**Why is Git useful for a team of developers?**  <br/>So everyone can work on the same project simultaneously. 
+
+
+**How do you create a new Git repository for a project locally?**  
+Open git-hub and hit the + button in the top left and create a anew repo by chosing where you want the repo to be on your computer. 
+
+**How do you create it on Github?**  <br/>You click the "+" link next to your user and click New repository.
+
+
+**How do you commit changes?**  <br/>You hit the commit button at the bottom of and edited page and make sure you are committing it to the right spot.
+
+
+**What is the difference between staging and committing changes?** <br/>Staging is seeing what it will look like before you commit it and take the snapshot.   
+
+
+**What is the difference between committing your changes and pushing them to Github?**  <br/>Committing just saves them to that version pushing them to Github to a remote repository so that others can view them. 
+
+
+**What is the command to check the status of your current repo in git?**  <br/>Git status. 
+
+
+**What is the command to see the history of your previous commits (from the command line)?**  <br/>What are a few interesting variations (sets of options)?  Git log. 
+
+
+**How can you look through your historical commits on the Github website?**  <br/>Git log dir/ 
```

Comment:
Nope.  In this case, on Github.com, you can click the link that says "35 Commits" (or whatever number) just above the file browser at the root of your project.